data-preprocessing/video_download/README.md

```diff
@@ -0,0 +1,9 @@
+This python script reads video links from a given sheet in an excel document, 
+downloads the videos and saves the videos to a google drive folder
+K.T Yeboah | 27-02-2023| 11:44am
+
+The instruction file (ins.json file) contains input variable parameters necessary to run the script.
+To run the script, first set the necesary parameters in the instruction file (ins.json) and save.
+The credentials.json file is a service file containing API keys, IDs and authorization credentials to the google drive folder in use. 
+To obtain a service file that points to a particular google drive folder, visit google cloud storage console (https://cloud.google.com/)
+Thereafter, invoke the python interpretor on the command line and add the name of the script=> py nsmq_vid.py
```

Comment:
suggesting a name change for the script to match the action. maybe `nsmq_vid_processing`?

Also the script is in the `video_download` folder but the job of the script is to download from youtube and then upload to G-Drive. So it's doing a bit more than the name suggests.

Would suggest go with a generic folder name so you can have different scripts in there instead of one script per folder. Maybe something like `video_preprocessing`, etc. The README can then have a section on each script in that folder explaining what it does.